tests/test_metrics_calibration.py

```diff
@@ -31,12 +33,108 @@ def test_sharpness_on_test_set(supply_test_set):
     assert np.abs(sharpness(test_std) - 0.648074069840786) < 1e-6
 
 
-def test_get_proportion_lists_on_test_set(supply_test_set):
-    """Test get_proportion_lists on the test set for some dummy values."""
-    test_exp_props, test_obs_props = get_proportion_lists(
-        *supply_test_set, num_bins=100, recal_model=None
+def test_root_mean_squared_calibration_error_on_test_set(supply_test_set):
+    """Test root mean squared calibration error on some dummy values."""
+    test_rmsce_nonvectorized = root_mean_squared_calibration_error(
+        *supply_test_set,
+        num_bins=100,
+        vectorized=False,
+        recal_model=None,
+        prop_type="interval"
     )
-    assert len(test_exp_props) == len(test_obs_props)
+    test_rmsce_vectorized = root_mean_squared_calibration_error(
+        *supply_test_set,
+        num_bins=100,
+        vectorized=True,
+        recal_model=None,
+        prop_type="interval"
```

Comment:
Do you think we also need a test for rms calibration error on `prop_type="quantile"`?